Alignment/OfflineValidation/interface/PVValidationHelpers.h

```diff
@@ -13,6 +13,22 @@
 
 namespace PVValHelper {
 
+  // helper logarithmic bin generator
+
+  template <typename T, size_t N>
+  std::array<T, N+1> makeLogBins(const double min, const double max) {
+    const double minLog10 = std::log10(min);
+    const double maxLog10 = std::log10(max);
+    const double width = (maxLog10-minLog10)/N;
+    std::array<T, N+1> ret;
+    ret[0] = std::pow(10, minLog10);
+    const double mult = std::pow(10, width);
+    for ( size_t i=1; i<=N; ++i) {
+      ret[i] = ret[i-1]*mult;
+    }
+    return ret;
+  }
+
```

Comment:
shouldn't `double` be replaced with `T` here?

Reply:
Hi @ghellwig, fixed in https://github.com/cms-sw/cmssw/pull/21078/commits/df904d2b7434a5860868e632707357550e6f9a1e

Reply:
Hi @mmusich, this actually applies to all occurences of `double`.

Reply:
or do you want to ensure that `double` is used internally?

Reply:
not really, everything is template type now.

Reply:
ok, thanks
not sure, if it should be `const T&` in the function argument, but for the types it is intended for this should not really matter